rmgpy/data/thermo.py

```diff
@@ -485,6 +500,47 @@ def bicyclicDecompositionForPolyring(polyring):
         bicyclicsMergedFromRingPair.append(mergedRing)
 
     return bicyclicsMergedFromRingPair, ringOccurancesDict
+
+def splitBicyclicIntoSingleRings(bicyclic_submol):
+    """
+    Splits a given bicyclic submolecule into two individual single 
+    ring submolecules (a list of `Molecule`s ).
+    """
+    SSSR = bicyclic_submol.getDeterministicSmallestSetOfSmallestRings()
+
+    return [convertRingToSubMolecule(SSSR[0])[0], 
+                convertRingToSubMolecule(SSSR[1])[0]]
+
+def saturateRingBonds(ring_submol):
+    """
+    Given a ring submolelcule (`Molecule`), makes a deep copy and converts non-single bonds 
+    into single bonds, returns a new saturated submolecule (`Molecule`)
+    """
+    from rmgpy.molecule.molecule import Molecule, Bond
```

Comment:
These have already been imported at the top of this file.

Reply:
they are removed now